FISH-5 Document renew-self-signed-certificates (#35)
payara/Payara-Enterprise-Documentation: docs/modules/ROOT/pages/documentation/payara-server/server-configuration/certificate-management.adoc

test_cert
----

[[renew-self-signed-certificates]]
== `renew-self-signed-certificates`

Usage::

> `asadmin> renew-self-signed-certificates

Reviewer: ```suggestion
`asadmin> renew-self-signed-certificates`
```

Aim::

> The renew-self-signed-certificates subcommand renews a self-certificate using the old distinguished name and alias.

Reviewer: ```suggestion
The `renew-self-signed-certificates` subcommand renews a self-signed certificate using the old distinguished name and alias used when it was previously generated using the `generate-self-signed-certificate` command.
```

Reviewer: Does the command renew a single certificate or all self-signed certificates in the keystore? Sometimes the documentation suggests it renews a single certificate and sometimes it suggests it will renew all certificates. 

If it renews a single certificate, how it's specified? I'm missing a way to specify the certificate alias.

If it renews all certificates, does it renew all of them or only those that are close to the expiration date? In this case, can we add an option to allow specifying a single alias to renew?

> The renew-self-signed-certificates subcommand renews all self-certificates in a keystore using the old distinguished name and alias.

Reviewer: ```suggestion
The renew-self-signed-certificates subcommand renews all self-signed certificates in a keystore using the old distinguished name and alias.
```

> This certificates are then placed in the key and trust store of target instance or listener replacing the old one.

Reviewer 2: ```suggestion
These certificates are then placed in the key and trust stores of the target instance or listener, replacing the old ones.
```

[[command-options-self]]
=== Command Options

[cols="3,1,5,1,1",options="header"]
|===
|Option
|Type
|Description
|Default
|Mandatory

|`--target`
|String

> |The name of the instance to renew the certificate of.

Reviewer: This is misleading. Instances have a keystore not specific certificates. Individual certificates are used for specific purposes (e.g. HTTP listener). Does this affect all certificates in the instance's keystore? Or just a specific certificate, e.g. admin HTTP listener certificate?
